src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/CollapsedTabList.js

```diff
@@ -2,29 +2,25 @@
 import React from 'react';
 import type {ChildrenArray, Element} from 'react';
 import classNames from 'classnames';
-import type {Skin} from './types';
 import CollapsedTab from './CollapsedTab';
 import collapsedTabListStyles from './collapsedTabList.scss';
+import type {TabType} from './types';
 
 type Props = {
     children: ChildrenArray<Element<typeof CollapsedTab> | false>,
-    skin: Skin,
+    type: TabType,
```

Comment:
guess we need to add a small hint to the `UPGRADE.md` that the `skin` prop wa replaced by a `type` prop